src/getHumanAge.test.js

```diff
@@ -7,4 +7,64 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should be converted when HumanAge is 0', () => {
+    expect(getHumanAge(0, 0))
+      .toStrictEqual([0, 0]);
+  });
+
+  test(`should be converted when HumanAge is less 
+        than 1 dog's and cat's year`, () => {
+    expect(getHumanAge(12, 12))
+      .toStrictEqual([0, 0]);
+  });
+
+  test(`should be converted when HumanAge is  
+        than 1 dog's and cat's year`, () => {
+    expect(getHumanAge(15, 15))
+      .toStrictEqual([1, 1]);
+  });
+
+  test(`should be converted when HumanAge is more 
+        than 1 dog's and cat's year but less than 2`, () => {
+    expect(getHumanAge(23, 23))
+      .toStrictEqual([1, 1]);
+  });
+
+  test(`should be converted when HumanAge is more 
```

Comment:
This test case is duplicated. It is the same as the test case in line 22. Please remove duplicate test cases to avoid redundancy and confusion.